jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/unmarshaller/UnmarshallerImpl.java

```diff
@@ -499,29 +499,27 @@ public void setAttachmentUnmarshaller(AttachmentUnmarshaller au) {
     /**
      * @deprecated since 2.0
      */
-    @Override
```

Comment:
add Deprecated annotation here (and other places you're modifying if it already contains doresponding javadoc tag like it is in this case)

Reply:
Added to both deprecated methods.